en/micro-integrator/docs/references/connectors/microsoft-azure-storage-connector/microsoft-azure-storage-connector-example.md

```diff
@@ -0,0 +1,286 @@
+
+Given below is a sample scenario that demonstrates how working with containers and blobs operations using the WSO2 Microsoft Azure Storage Connector.
+
+## What you'll build
+
+This example demonstrates how to use Microsoft Azure Storage connector to:
+
+1. Create a container (a location for storing employee details) in Microsoft Azure Storage account.
+2. Retrieve information about the created containers.
+3. Upload text or binary employee details (blob) in to the container.
+4. Retrieve information about the uploaded employee details(blob).
+5. Remove uploaded employee details (blob).
+6. Remove created container.
+
+All six operations are exposed via an API. The API with the context `/resources` has six resources  
+
+* `/createcontainer` : Creates a new container in the Microsoft Azure Storage account with the specified container name for store employee details.
+* `/listcontainer` : Retrieve information about the created containers from the Microsoft Azure Storage account.
+* `/adddetails`: Upload text or binary employee data (blob) and stored into the specified container.
+* `/listdetails` : Retrieve information about the added employee deta (blobs).
+* `/deletedetails` : Remove added employee data from the specified text or binary employee data (blob).
+* `/deletecontainer` : Remove created container in the Microsoft Azure Storage account.
+
+To know the further information about these operations please refer the [Microsoft Azure Storage connector reference guide](microsoft-azure-storage-connector-reference.md).
+
+> **Note**: Before invoke the API you need to create a **Storage Account** in **Microsoft Azure Storage account**, see [Azure Storage Configuration](microsoft-azure-storage-configuration.md) documentation.
+
+Following diagram shows the overall solution. User creates a container, stores some text or binary employee data (blob) into the container and then he receives it back. To invoke each operation user uses the same API. 
+
+<img src="/assets/img/connectors/MS-azure-storage-connector.png" title="Microsoft Azure Storage Connector" width="800" alt="Microsoft Azure Storage Connector"/>
+
+If you do not want to build this yourself, you can simply [get the project](#get-the-project) and run it.
+
+## Configure the connector in WSO2 Integration Studio
+
+Follow these steps to set up the ESB Solution Project and the Connector Exporter Project.
+
+{!references/connectors/importing-connector-to-integration-studio.md!}
+
+1. Right click on the created ESB Solution Project and select, -> **New** -> **Rest API** to create the REST API.
+
+2. Specify the API name as `MSAzureStorage` and API context as `/resources`. You can go to the source view of the XML configuration file of the API and copy the following configuration (source view).
```

Comment:
You can go to the XML configuration of the API (source view) and copy the following configuration.

Reply:
Updated.